homeassistant/components/camera/nest.py

```diff
@@ -32,6 +45,25 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                for structure, device in camera_devices]
     add_devices(cameras, True)
 
+    def service_handler(service):
+        """Handle service call."""
+        entity_ids = extract_entity_ids(hass, service)
+        entities = [entity for entity in cameras
+                    if entity.available and entity.entity_id in entity_ids]
+        for camera in entities:
+            if service.service == SERVICE_TURN_ON:
+                camera.is_recording = True
+            elif service.service == SERVICE_TURN_OFF:
+                camera.is_recording = False
+
+    hass.services.async_register(
+        DOMAIN, SERVICE_TURN_ON, service_handler,
```

Comment:
Prefix the service name with the platform name.

Reply:
camera.nest.turn_on?

Reply:
`camera.nest_turn_on`